components/DocsHelp.tsx

```diff
@@ -1,8 +1,16 @@
 import { useRouter } from 'next/router';
 import React, { FormEvent, useRef, useState } from 'react';
 
-export function DocsHelp() {
+interface DocsHelpProps {
+  markdownFile?: string;
+}
+
+export function DocsHelp({ markdownFile }: DocsHelpProps) {
   const router = useRouter();
+  const path = encodeURIComponent(router.pathname);
+  const targetUrl = markdownFile
+    ? `https://github.com/json-schema-org/website/blob/main/pages${router.asPath}.md`
```

Comment:
Why this reference to blogs here instead of the blog pages?

Reply:
it was just a single line code , should i shift it to blog(index.page.tsx) or remove the const targetUrl and use that directly in href instead of 
`href={targetUrl}`